install/linkstack-install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: Omar Minaya
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://linkstack.org/
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing dependencies"
+$STD apt-get install -y \
+    software-properties-common \
+    ca-certificates \
+    lsb-release \
+    apt-transport-https \
+    apache2
+    unzip
+msg_ok "Installed dependencies"
+
+PHP_VERSION=8.2
+PHP_MODULE=sqlite3,mysql,gd,curl,mbstring,zip,xml,bcmath,fileinfo
+PHP_APACHE=YES
+
+install_php
```

Comment:
```suggestion
PHP_VERSION="8.2" PHP_MODULE="sqlite3, mysql, fileinfo" PHP_APACHE="YES" install_php
```
I already fixed this, not sure why are you touching this again?